### ISSUE TITLE
[RFE] Exclude dictionary files from backup

### ISSUE THREAD

**Issue by genodeftest** [rdoeffinger/Dictionary#105]
When taking a backup from my smartphone (`adb backup -apk -all`) it will grow quite large. One reason for that is that quickdic does include all its dictionary files in the backup.

You may want to use [this Android API](https://developer.android.com/guide/topics/data/autobackup#IncludingFiles) to exclude the `/apps/de.reimardoeffinger.quickdic/r/` (webkit textures, cache) and `/apps/de.reimardoeffinger.quickdic/ef/` (dictionary contents) folders.

**Comment by rdoeffinger** [comment]
Unless/until Android adds some kind of configurability to the backup process this is not an issue.
Excluding the dictionaries assumes they can easily obtained again.
However this isn't generally true.
Users might have custom dictionaries, they might currently reside in a country that blocks github and VPNs, they might not have an internet connection at home so that they can only get dictionaries from friends or a library, or might only have a metered connection.
Thus including the dictionaries in the backup seems like the less wrong solution to me.

**Comment by genodeftest** [comment]
I don't think that many users will have custom dictionaries and thus disagree on the default. Anyhow, I see your point.

I agree that Android is missing important configurability here though.

Anyway, would it be possible to exclude the `/r/` folder?
